doc/content/gateways/lora-basics-station/cups.md

```diff
@@ -49,10 +90,11 @@ Upload the `.pem` file in your gateway as the CUPS Server Certificate / CUPS Tru
 
 This is a file which {{% tts %}} uses to verify the identity of your gateway.
 
-Use the following command to create a file called `cups.key` with the `<gateway-api-key>` you created above.
+Use the following commands to create a file called `cups.key`, replacing `"your-cups-api-key"` with the CUPS API key you created above.
 
 ```bash
-echo "Authorization: Bearer <gateway-api-key>" | perl -p -e 's/\r\n|\n|\r/\r\n/g'  > cups.key
+$ export CUPS_KEY="your-cups-api-key"
+$ echo "Authorization: Bearer $CUPS_KEY" | perl -p -e 's/\r\n|\n|\r/\r\n/g'  > cups.key
 ```
 
 > The above command creates a file called `cups.key`, terminated with a Carriage Return Line Feed (`0x0D0A`) character. Upload this file in your gateway as the CUPS key.
```

Comment:
Maybe add a note in the end that if successful, the CUPS server will send the LNS URI (`gatewayserveraddress`), LNS trust and the LNS API Key to the gateway and it will automatically attempt to connect to the LNS Server.